wagtail/admin/tests/pages/test_bulk_actions/test_bulk_move.py

```diff
@@ -321,3 +321,34 @@ def hook_func(request, action_type, pages, action_class_instance):
             Page.objects.get(id=self.test_page_b_1.id).get_parent().id,
             self.section_b.id,
         )
+
+    def test_bulk_move_with_restricted_parent_page_types(self):
```

Comment:
@th3hamm0r I finally got to looking into this.

I am afraid I can't replicate the issue with latest main. Is it an issue limited to 4.x?

for example, I added only this test on `main` and ran `py runtests.py wagtail.admin.tests.pages.test_bulk_actions.test_bulk_move.TestBulkMove.test_bulk_move_with_restricted_parent_page_types` and passes without an issue.

Reply:
Yeah, you've already fixed it with #10362 ;)
I've mentioned it in the PR description above, but I should have emphasized it more, sry...
So this test fails with <= 5.0 (your fix has been merged into main for 5.1).

The question is, should we still keep my test?

Reply:
Let's leave it as it is another dimension to #10362.

@gasman #10362 and ##10473 do affect 4.1 and it somehow looks like people just now are stumbling upon it. I respect your reasoning in https://github.com/wagtail/wagtail/pull/10362#issuecomment-1520360164, and at the same time wonder whether this is grounds enough to backport to 4.1 since it is our LTS for some time still